keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2840,135 +2899,262 @@ GA:
 
-* **Are there any tests for feature enablement/disablement?** No. Manual tests
-  will be run before switching feature gate to beta.
+The feature will be restored.
+
+###### Are there any tests for feature enablement/disablement?
+
+No. Manual tests will be run before switching feature gate to beta.
 
 ### Rollout, Upgrade and Rollback Planning
 
-* **How can a rollout fail? Can it impact already running workloads?** A
-  misconfiguration could cause apiserver requests to be rejected, which could
-  have widespread impact such as: (1) rejecting controller requests, thereby
-  bringing a lot of things to a halt, (2) dropping node heartbeats, which may
-  result in overloading other nodes, (3) rejecting kube-proxy requests to
-  apiserver, thereby breaking existing workloads, (4) dropping leader election
-  requests, resulting in HA failure, or any combination of the above.
-
-* **What specific metrics should inform a rollback?** An abnormal spike in the
-  `apiserver_flowcontrol_rejected_requests_total` metric should potentially be
-  viewed as a sign that kube-apiserver is rejecting requests, potentially
-  incorrectly. The `apiserver_flowcontrol_request_queue_length_after_enqueue`
-  metric getting too close to the configured queue length could be a sign of
-  insufficient queue size (or a system overload), which can be precursor to
-  rejected requests.
-
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  No. Manual tests will be run before switching feature gate to beta.
-
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
-  fields of API types, flags, etc.?** Yes, `--max-requests-inflights` will be
-  deprecated in favor of APF.
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+A misconfiguration could cause apiserver requests to be rejected,
+which could have widespread impact such as: (1) rejecting controller
+requests, thereby bringing a lot of things to a halt, (2) dropping
+node heartbeats, which may result in overloading other nodes, (3)
+rejecting kube-proxy requests to apiserver, thereby breaking existing
+workloads, (4) dropping leader election requests, resulting in HA
+failure, or any combination of the above.
+
+###### What specific metrics should inform a rollback?
+
+An abnormal spike in the
+`apiserver_flowcontrol_rejected_requests_total` metric should
+potentially be viewed as a sign that kube-apiserver is rejecting
+requests, potentially incorrectly. The
+`apiserver_flowcontrol_request_queue_length_after_enqueue` metric
+getting too close to the configured queue length could be a sign of
+insufficient queue size (or a system overload), which can be precursor
+to rejected requests.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+No. Manual tests will be run before switching feature gate to beta.
```

Comment:
We were doing them, but I can't remember the exact setup as it was 2 years ago or so.

Let's say that "Manual test were run during promotion to Beta in 1.20